fedimint-server/src/consensus/mod.rs

```diff
@@ -70,6 +69,7 @@ pub async fn run(
     let mut global_dbtx = db.begin_transaction().await;
     apply_migrations_server_dbtx(
         &mut global_dbtx.to_ref_nc(),
+        Arc::new(ServerDbMigrationContext),
```

Comment:
The real migration context specific for server side is injected from `fedimint-server`.

Reply:
What do you mean by "real"?

Reply:
The "implementation" - one that actually implements getting history items (or any apis we would like to expose to the modules).